### PR TITLE
POWR-1750 - Media losing group content relationship when created inline

### PR DESCRIPTION
**Pull request by jookyg**
* Implements hook_entity_insert to manually add a new media entity to a group if it is created in the context of an inline entity form. Works with all media types.
* Reverts temporary changes to entity browsers as workaround while bug was fixed.

### CODE REVIEW

File: web/modules/custom/portland/modules/portland_groups/portland_groups.module [hunk 1]
```diff
@@ -53,6 +53,39 @@ function _portland_groups_inline_entity_populate_field_display_groups(&$form, &$
...
+
+/**
+ * Manually assigns a media entity to a group as group content.
+ * The group is identified by the first delta in the media's field_display_groups.
+ * This should only happen in the context of an inline entity form.
+ */
+function _portland_groups_assign_media_to_group($entity) {
+
+  // only do this for inline entity brower
+  $current_path = \Drupal::service('path.current')->getPath();
```
Comment (joshuami) [inline web/modules/custom/portland/modules/portland_groups/portland_groups.module]: How expensive will it be to call a function like this on every page? 🤔

Comment (jookyg) [reply web/modules/custom/portland/modules/portland_groups/portland_groups.module]: I really have no idea. It seems to be a pretty commonly used function, so I assumed it was negligible. It doesn't get called on every page, just whenever a media item is created. I can look into getting the current path/url using php server var.

Comment (jookyg) [reply web/modules/custom/portland/modules/portland_groups/portland_groups.module]: If you're concerned about the performance of that call, I can replace it with $_SERVER['REQUEST_URI'] without much hassle. Though I would expect that path/url operations are pretty lightweight and don't use any db calls or heavy string manipulation.

Comment (joshuami) [reply web/modules/custom/portland/modules/portland_groups/portland_groups.module]: No need to rewrite. I was just musing.

**Pull request merged**